Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/library/alpine:3.21.3
+LABEL source_repository="https://github.com/sapcc/alertmagnet"
+
+RUN apk --update add python3 bash git py3-setuptools
+RUN apk upgrade -U 
+
+RUN git clone https://github.com/sapcc/alertmagnet.git
+WORKDIR /alertmagnet
```

Comment:
Why no `COPY . /alertmagnet` together with `.dockerignore` to exclude any local venvs? 🧐

Reply:
If this later runs in a pipeline I rather don't want to explicitly checkout the gh repository beforehand to do the adding.
Also it prohibits adding anything critical to the image which might be floating around locally and has been forgotten in the `.dockerignore`.

Reply:
But then testing things locally is near impossible and the concourse pipeline is also more luck based than on the pinned resources.

Reply:
Locally you can link your repository folder to overwrite `/alertmagnet` and have everything in for testing.